java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/ConnectFailedException.java

```diff
@@ -2,11 +2,9 @@
 
 package com.zeroc.Ice;
 
-/** This exception indicates connection failures. */
+/** The exception that is thrown when a connection establishment fails. */
```

Comment:
"The exception that is thrown when ..." is the standard way we document exceptions.